projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss

```diff
@@ -419,9 +483,18 @@
         pointer-events: none;
         box-shadow: none;
 
-        &%igx-button--flat {
+        &%igx-button--flat,
+        &%igx-button--outlined {
             background: transparent !important;
         }
+
```

Comment:
We can make the code so that there is no need of "!important" rules

```
%igx-button--disabled {
        pointer-events: none;
        box-shadow: none;

        &%igx-button--flat,
        &%igx-button--outlined,
        &%igx-button--raised,
        &%igx-button--fab,
        &%igx-button--icon {
            color: --var($theme, 'disabled-color');
        }

        &%igx-button--raised,
        &%igx-button--fab {
            background: --var($theme, 'disabled-background');
        }

        &%igx-button--flat,
        &%igx-button--outlined {
            background: transparent;
        }

        &%igx-button--outlined {
            border-color: --var($theme, 'outlined-disabled-outline-color');
        }

        &:focus {
            box-shadow: none;
        }
    }
```

Reply:
@desig9stein Thanks!